package.json

```diff
@@ -7,17 +7,20 @@
     "url": "https://github.com/openfisca/openfisca-doc.git"
   },
   "scripts": {
+    "prebuild": "npm run clean",
     "build": "gitbook build",
-    "clean": "rm -rf _book",
+    "clean": "del _book",
```

Comment:
No, `del` is a cross-platform abstraction layer over `rm`. It has been properly declared as a `devDependency` (`del-cli`). Have you run `npm install`? :wink: